README.md

```diff
@@ -45,6 +46,12 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 ## Others
 
+--
+
+## Testing
+
+- [How to Start a Career in Testing? ](https://juniorlearnstocode.blogspot.com/2020/04/how-to-start-career-in-testing.html/)
+
```

Comment:
Can you please add a horizontal like "--" after Testing? This is just to maintain the file format!

Reply:
Of course, I'll add it right away. Cheers!